implementation_guide/ig_management.rst

```diff
@@ -121,3 +121,97 @@ Manage your IG using GitHub
 
 The GitHub webhook allows managing your Implementation Guide, without using the editor itself. You can find more information on how to set this up in the `GitHub integration documentation <../data_governance_and_quality_control/simplifierGithub.html#github-webhook-to-manage-implementation-guides>`_.
 
+Pre-Requisites
+--------------
+
+Below are the necessary pre-requisites needed for managing an implementation guide in Github
+
+- Create An account on Simplifier.net with the ability to create an Implementation Guide
+
+- Create a Github account
+
+- Install Git on your local computer
+
+----------------------------------------------------------------------------------------------
+
+1. Introduction
+---------------
+
+This guide will provide step by step instructions on managing your implementation Guide using Github.
+
+2. Setting up a Github Repository
+---------------------------------
+
+2.1 Create a Repository
+
+   1. Log into your Github account.
+
+   2. Click on the + icon in the top right corner and select New repository
+
+   3. Enter a repository name, description, and choose visibility (Public or Private)
+
+   4. Optionally, you can add a README file, a .gitignore file as well as the ability to choose a license.
+
+   5. Click Create Repository
+
+   .. image:: ../images/GithubNewRepository.png
+      :scale: 75%
+
+   .. image:: ../images/GithubCreateNewRepository.png
+      :scale: 75%
+
+2.2 Clone the Repository on your Local computer
+
+   - Copy your clone using the web URL from your Github Repository
+
+   .. image:: ../images/GithubCloneWebURL.png
+      :scale: 75%
+
+   - Open Git Bash
+
+   - CD to whichever directory you want to clone your repository into
+
+   - Clone the repository using the commands: 
+
+   .. image:: ../images/cmdtoclonerepository.png
+      :scale: 75%
+
```

Comment:
I think a lot of the people working with Git and who would want to manage their IG in Git, know how Git works, so I think it is better to leave out these steps. You can always keep them for your personal documentation of course.